k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-server/templates/cortx-server-pod.yaml

```diff
@@ -66,9 +66,10 @@ spec:
         args:
           - -c
           {{- if .Values.cortxserver.machineid.value }}
-          - /opt/seagate/cortx/provisioner/bin/cortx_deploy -n $MACHINE_ID -f /etc/cortx/solution -c yaml:///etc/cortx/cluster.conf
+          - set -x;
+            /opt/seagate/cortx/provisioner/bin/cortx_deploy -n $MACHINE_ID -f /etc/cortx/solution -c yaml:///etc/cortx/cluster.conf;
```

Comment:
Same question, is the `;` necessary?

Reply:
Semicolons included for consistency with multi-line entrypoints and prevent future issues with updates to entrypoints.